locales/ES_es.json

```
@@ -153,13 +153,13 @@
 			},
 			{
 				"name": "Stary",
-				"caption": "3DS dev and NEX dissector",
+				"caption": "desarrollador de 3DS y disector NEX",
```

Could you fix the capitalization? It's triggering my OCD lol
```suggestion
				"caption": "Desarrollador de 3DS y disector NEX",
```